content/docs/components/badge/theming.mdx

```diff
@@ -3,4 +3,182 @@ id: badge
 scope: theming
 ---
 
-TODO
+
+The `Badge` component is a single part component. All of the styling is applied
+directly to the `span` element.
+
+> To learn more about styling single part components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-single-part-components)
+> page.
+
+## Theming properties
+
+The properties that affect the theming of the `Badge` component are:
+
+- `variant`: The visual variant of the badge. Defaults to `solid`.
+- `colorScheme`: The color scheme of the badge. Defaults to `gray`.
+- `size`: The size of the badge. Defaults to `md`.
```

Comment:
- The default variant is `subtle` (Check the properties in the [Badge Theme](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/badge.ts))
- The `Badge` component doesn't have a default size.

Reply:
Fixed it on https://github.com/chakra-ui/chakra-ui-docs/pull/1136/commits/dccb6e2c087de07214fb71ad052ea6cf5e93afcd